server/http_server.py

```
@@ -0,0 +1,493 @@
+"""HTTP server for the Trakt MCP server."""
+
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)